www/src/pages/en/introduction.md

```diff
@@ -33,13 +33,18 @@ The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t
 
 Kind of. We love the technologies that the T3 Stack includes but we do not believe that all of them are needed in every project. So we made `create-t3-app` to do one thing: _**Simplify complex boilerplate around the core T3 Stack tech without compromising the pieces modularity.**_
 
-We are selective about the packages we have included. This is NOT an all-inclusive template and we expect you to bring your own libraries as well.
+## T3 Axioms
 
-We don't add libraries that are as simple as `npm install zustand`: If you cut an issue asking us to add your preferred libraries, we will make fun of you.
+We'll be frank - this is an _opinionated project_. We share a handful of core beliefs around building and we treat them as the basis for our decisions.
 
-<p align="center">
-  <img
-    src="https://media4.giphy.com/media/3orieLHXgpfkKO9Iju/200.gif"
-    alt="Nelson from the Simpsons pointing and saying ha-ha in a mocking tone"
-  />
-</p>
+### Solve Problems
+
+It's easy to fall in the trap of "adding everything" - we explicitly don't want to do that. Everything added to `create-t3-app` should solve a specific problem that exists within the core technologies included. This means we won't add things like state libraries (`zustand`, `redux`) but we will add things like NextAuth.js and integrate Prisma and tRPC for you.
+
+### Bleed Responsibly
+
+We love our bleeding edge tech. The amount of speed and, honestly, fun that comes out of new shit is really cool. We think it's important to bleed responsibly, using riskier tech in the less risky parts. This means we wouldn't ⛔️ bet on risky new database tech (SQL is great!). But we happily ✅ bet on tRPC since it's just functions that are trivial to move off.
+
+### Typesafety Isn't Optional
+
+Two of the three T's are typesafe (Typescript, tRPC). We take typesafety seriously in these parts. Any decision that compromises the full-stack, typesafe nature of `create-t3-app` is a decision that should be made in a different project. The stated goal of `create-t3-app` is to provide the quickest way to start a new fullstack, type safe web application.
```

Comment:
Two of the three T's are regarding typesfety ...

Reply:
probably no apostrophe on the T too

Reply:
rewrote this paragraph, here and in readme